defaults/main.yml

```diff
@@ -18,7 +18,7 @@ consul_ui_server_port: 80
 consul_install_nginx: true
 consul_install_nginx_config: true
 consul_enable_nginx_config: true
-consul_service_state: restarted
+consul_service_state: started
```

Comment:
It's a default value, so it should be set to something and then overwritten if necessary.